noir-projects/aztec-nr/aztec/src/authwit/account.nr

```diff
@@ -18,6 +18,15 @@ impl<Context> AccountActions<Context> {
     }
 }
 
+pub struct FeePaymentMethodEnum {
```

Comment:
```suggestion
/// Noir representation of the `AccountFeePaymentMethodOptions` in the TypeScript `entrypoint` package.
pub struct FeePaymentMethodEnum {
```

Note that in js we call this struct `AccountFeePaymentMethodOptions`, it'd be nice for them to share the name.

Reply:
Renamed both the same for consistency, changed `FEE_JUICE` option to `CURRENT_FEE_JUICE_BALANCE`. WDYT?

Reply:
Also shuffled the options around so EXTERNAL gets assigned value 0...just in case TS messes up. There shouldn't be a difference. but lines up better with "do nothing unless explicitly asked"

Reply:
I think 'preexisting' makes it much more obvious that it is wrong if you're thinking about claiming and using it immediately. 'current' is a bit vague - what'd be not current? Surely not future, so past? It turns out it actually _is_ the past balance, not the 'current' balance. Idk it gets into annoying timing semantics. I prefer 'preexisting' personally.

Reply:
Well, it's current because you haven't claimed YET. It will happen in the same tx, but strictly speaking you're paying with a combination of your current (right now) balance and the sum of the claim that will be resolved later in public

Reply:
🤷‍♂️